test/parallel/test-repl-persistent-history.js

```diff
@@ -178,6 +179,21 @@ const tests = [
     test: [UP],
     expected: [prompt]
   },
+  {
+    before: function before() {
+      if (!common.isWindows) {
```

Comment:
Yes, because like the test case before it, it will just act like an empty file on other platforms.